MANIFEST.in

```diff
@@ -0,0 +1,11 @@
+include .coveragerc
+include CHANGELOG.md
+include COPYING.txt
+include Makefile
+include README.md
+
+graft gneiss
+
```

Comment:
Do you think that the IPython notebooks should be pip installable?  I don't
have strong feelings about this.

On Tue, Aug 30, 2016 at 10:07 AM, Daniel McDonald <notifications@github.com>
wrote:

> In MANIFEST.in
> <https://github.com/biocore/gneiss/pull/44#discussion_r76837301>:
>
> > @@ -0,0 +1,11 @@
> > +include .coveragerc
> > +include CHANGELOG.md
> > +include COPYING.txt
> > +include Makefile
> > +include README.md
> > +
> > +graft gneiss
> > +
>
> do you want to include ipynb?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/biocore/gneiss/pull/44/files/6870b42da40d27dfc6a713ae20d12ccd4ff4961a#r76837301>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AD_a3QA4hGEHDWfUCBUK9hJTBJVpoi_fks5qlGNJgaJpZM4JuYBq>
> .
>